test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/ZookeeperK8sService.java

```diff
@@ -239,7 +239,7 @@ private V1Deployment getDeployment() {
                 .put("kind", CUSTOM_RESOURCE_KIND)
                 .put("metadata", ImmutableMap.of("name", deploymentName))
                 .put("spec", ImmutableMap.builder().put("image",  getImageSpec(DOCKER_REGISTRY + PREFIX + "/" + ZOOKEEPER_IMAGE_NAME, PRAVEGA_ZOOKEEPER_IMAGE_VERSION))
-                                         .put("size", clusterSize)
+                                         .put("replicas", clusterSize)
```

Comment:
This change seems implies that the newer zookeeper operator is not backward compatible. Is that the case ? if yes could share the zookeeper-operator version above which this change will work.

Reply:
The changes are backward compatible as `replicas` filed is available in previous zookeeper  versions as well.